tools/c7n_azure/c7n_azure/resources/key_vault_keys.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2018 Capital One Services, LLC

```

Comment:
This project uses contributor license agreements from apache foundation, which do not use copyright assignment. Hence these files should reflect the entity/organization of the author to be accurate.

Reply:
Should we just omit all copyright moving forward? Not sure they are needed? 

Seems this addition to the first sentence of the license header is common:
```
Licensed to XXX under one or more contributor license agreements; and to You under the Apache License, Version 2.0. 
```

Reply:
let's table this for now, there's a discussion happening about what we should be doing, ie switch to project authors as a generic term here, each contributor putting in copyright info on new files, etc. its a discussion we can wrap up at the pycon sprint.